src/components/modules/epics/ContributionsEpicReminder.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import { css } from '@emotion/core';
+import { from } from '@guardian/src-foundations/mq';
+import { palette } from '@guardian/src-foundations';
+import {
+    ContributionsEpicReminder,
+    ContributionsEpicReminderProps,
+} from './ContributionsEpicReminder';
+
+const containerStyles = css`
+    margin: 3em auto;
+    padding: 0 10px;
+    max-width: 620px;
+
+    ${from.mobileLandscape} {
+        padding: 0 20px;
+    }
+`;
+
+const backgroundStyles = css`
+    background-color: ${palette.neutral[97]};
+`;
+
+export const EpicDecorator = (Story: Story): JSX.Element => (
+    <div css={containerStyles}>
+        <div css={backgroundStyles}>
+            <Story />
+        </div>
+    </div>
+);
```

Comment:
If we like this, I'll happily refactor our other epic stories to use this. I didn't really like that the story version of the epic just fills the whole display, because that's not how it looks on the size. This makes it so the epic is at least the same size as it would be dotcom e.g 

<img width="1396" alt="Screenshot 2021-03-25 at 14 46 00" src="https://user-images.githubusercontent.com/17720442/112492071-d9869580-8d78-11eb-9a19-d1c9eba4a862.png">